### ISSUE TITLE
[a11y] Talkback doesn't announce the number of list items from the menus

### ISSUE THREAD

**Issue by lobontiumira** [mozilla-mobile/focus-android#6132]
## Steps to reproduce
1. Turn talkback/screen reader on.
2. Open Focus, and open the three-dot menu using talkback gestures.
3. Listen what the screen reader announces.

### Expected behavior
The screen reader announces the first item, and the number of items from the list.

### Actual behavior
The screen reader announces the menu as "Pop-up window", and reads only the first item, that is selected.

### Device information
* Android device: Google Pixel (Android 10)
* Focus version: Beta 96.0.0-beta.5, 12/27 Focus Nightly
* The Settings options aren't read as list items either.

**Comment by lobontiumira** [comment]
Hello!
I've tested again today on the latest Focus Nightly build, Fenix Nightly, and Chrome, with Google Pixel (Android 10).
On Fenix, and Focus, the talkback assistant doesn't announce the number of items from the main menu, but on Chrome, the same assistant announces the name of the item, and the "1 of 17 items".

**Comment by lobontiumira** [comment]
I've retested again today, with the latest Talkback version 12.2, on both Focus Nightly 102.0a1, Fenix Nightly 102.0a1, and Chrome with several devices:
* Google Pixel (Android 10) - only on Chrome, the page's main menu is read as "1 of 17, in list 17 items",
* Xiaomi Mi8 Lite (Android 10) - only on Chrome, the page's main menu is read as "1 of 17, in list 17 items",
* Oppo Reno 6 (Android 12) - Talkback doesn't read the page's main menu as in list on neither of the browsers,
* Sony Xperia Z5 Premium (Android 7.1.1) - only on Chrome, the page's main menu is read as "1 of 17, in list 17 items".

I'll close this ticket.
